lib/src/navigation/routes/bindings_interface.dart

```diff
@@ -16,14 +18,20 @@ abstract class Bindings {
 /// GetPage(
 ///   name: '/',
 ///   page: () => Home(),
-///   binding: BindingsBuilder(() => Get.put(HomeController())),
+///   // This might cause you an error.
+///   // binding: BindingsBuilder(() => Get.put(HomeController())),
+///   binding: BindingsBuilder(() { Get.put(HomeController(); })),
```

Comment:
why this may cause errors?

Reply:
Dart has been having some problems with arrow functions recently.
I also experienced errors using arrow functions in the last few days, which makes no sense, but in the dev version of Flutter (which points to the dev version of dart) this is happening frequently.
I don't know if this needs to be documented, as this problem will certainly be fixed in the dart

Reply:
this is indeed weird, and I agree with you, although is a nice thing, this does not need to be documented here

Reply:
I can remove the comment, but we should wait til this gets solved. As long as u use lazyPut() or the new factory constructor `BindingsBuilder.put` for 1 reference, you should be fine.